### PR TITLE
LPS-148619

### PR DESCRIPTION
**Pull request by arthurchan35**
[LPS-148619](https://issues.liferay.com/browse/LPS-148619)

### CODE REVIEW

**Comment by arthurchan35** [comment]
ci:forward

**Comment by arthurchan35** [comment]
From https://github.com/liferay-appsec/liferay-portal/pull/583

**Comment by arthurchan35** [comment]
@weller0246 functional tests have sf problem, can you fix and comment ci:forward?

**Comment by weller0246** [comment]
ci:test:bundle:norebase

**Comment by weller0246** [comment]
ci:test:oidc

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase [hunk 1]
```diff
@@ -455,6 +455,69 @@ definition {
 		}
 	}
 
+	test OpenIDConnectWithGoogleCustomParameters {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase:458]: Please add ticket description and priority annotations

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase [hunk 2]
```diff
@@ -455,6 +455,69 @@ definition {
...
+		property test.name.skip.portal.instance = "OpenIDConnect#EnabledOpenIDConnectOnSystemLevel";
+
+		var customTokenRequestParameters = "resource=https://contacts.google.com";
+		var customAuthorizationRequestParameters = "access_type=offline";
+		var openIdConnectClientId = PropsUtil.get("google.client.id.2");
+		var openIdConnectClientSecret = PropsUtil.get("google.client.secret.2");
+		var userEmailAddress = PropsUtil.get("email.address.2");
+		var userPassword = PropsUtil.get("email.password.2");
+
+		task ("Enable DEBUG level logs for OpenIDConnect") {
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/OpenIDConnect.testcase:469]: As we discussed we could use the GWT prefixes in our task names like here: https://github.com/liferay/liferay-portal/blob/master/modules%2Fapps%2Fcommerce%2Fcommerce-product-test%2Fsrc%2FtestFunctional%2Ftests%2FCPCommerceOrderTypes.testcase#L294-L312

cc @JoyceWang08211

**Comment by arthurchan35** [comment]
ci:test:oidc

**Comment by arthurchan35** [comment]
ci:forward

**Pull request closed without merging**